tests/output/ConsoleInteractionTest.py

```diff
@@ -714,7 +718,9 @@ def test_show_bear_with_content(self):
                              "Optional, defaults to 'None'."
                              ')\n\n'
                              '  Can detect:\n   * Formatting\n\n'
-                             '  Can fix:\n   * Formatting\n\n')
+                             '  Can fix:\n   * Formatting\n\n'
+                             '  Path:\n'
+                             '   ' + repr(TestBear.file_path) + '\n\n\n')
```

Comment:
Why the change from two `\n\n` to three `\n\n\n` at the end? :(

I think only one `\n` at the end of the `console_printer.print` in `ConsoleInteraction.py`.

LGTM otherwise :)